controller/cache/info.go

```diff
@@ -146,6 +146,13 @@ func populateIngressInfo(un *unstructured.Unstructured, res *ResourceInfo) {
 						tlshost := tlsline["host"]
 						if tlshost == host {
 							stringPort = "https"
+							continue
+						}
+						tlshosts := tlsline["hosts"].(map[string]interface{})
```

Comment:
Ops, TestGetIngressInfoWithHost crashed with a panic. 
Please add a check to avoid panic here.